cimsparql/line_queries.py

```diff
@@ -218,3 +218,55 @@ def borders_query(
         where_list.append(sup.combine_statements(f"FILTER (!regex({name}, 'HVDC'))", group=True))
 
     return sup.combine_statements(sup.select_statement(variables), sup.group_query(where_list))
+
+
+def exchange_query(
+    cim_version: int,
+    region: Union[str, List[str]],
+    nodes: str,
+    ignore_hvdc: bool,
+    with_market_code: bool,
+    market_optional: bool,
```

Comment:
Here we have both `with_market_code` and `market_optional`. Is it nessecary? If `with_market_code` can it just always be `optional` in the query?